README.md

```diff
@@ -21,7 +21,7 @@ If you are sideloading this provider (i.e. not getting this via the Terraform st
       required_providers {
         onelogin = {
           source  = "onelogin.com/onelogin/onelogin"
-          version = "0.1.10"
+          version = "0.2.0"
```

Comment:
This bit of the doc is specifically around locally testing changes made while developing the provider. I am more than happy to take this change, but can you also update this version: https://github.com/onelogin/terraform-provider-onelogin/blob/develop/GNUmakefile#L14